test/integration/roles/test_ec2_eip/tasks/main.yml

```diff
@@ -152,7 +152,7 @@
 - name: assert new EIP was assigned
   assert:
     that:
-      - '"failed" not in result'
```

Comment:
Well, the task didn't fail, otherwise we wouldn't be here. We are in fact checking if the key exists (as it should always be). The only reason I added it, is because all the other tasks had it, so I assume this was an oversight.

Checking if the task did not fail is of no use IMO.